### ISSUE TITLE
Add GitLab proof

### ISSUE THREAD

**Issue by jayvdb** [keybase/proofs#85]
https://gitlab.com/

**Comment by jayvdb** [comment]
A workaround is to use a GitLab Pages website as a proof.
But it could be a group, not a user.  A GitLab API lookup is needed to distinguish between them.

**Comment by nmarley** [comment]
Please add this - there should be huge demand in the wake of MS takeover. I personally would LOVE to be able to prove who I am on GitLab as I migrate away from GitHub.

**Comment by bkcsoft** [comment]
GitLab has snippets, could be done similar to the BitBucket proof https://github.com/keybase/proofs/pull/18. I would also like this very much :)

**Comment by yan12125** [comment]
The GitHub proof is done via searching the gists via the API `https://api.github.com/users/#{username}/gists`. On Gitlab, it's possible to do the same thing on the website (e.g., from https://gitlab.com/users/yan12125/snippets), but seems there's no API to do the same thing (https://gitlab.com/gitlab-org/gitlab-ce/issues/47489).

**Comment by MyriaCore** [comment]
I gotta bump this up. I use gitlab all the time, it's frustrating to have my keybase not link it. I'd love to be able to add it myself in a pull request, but because of Issue #102 learning the way proofs are added would take way more time than I have ATM.

**Comment by shareme** [comment]
yes can we have this might be at first best to do it via the user website gitlab pages features

**Comment by yocorn** [comment]
+1

**Comment by TomBaxter** [comment]
Not sure if this helps but, [This issue](https://gitlab.com/gitlab-org/gitlab-ce/issues/41888) was closed with this [commit](https://gitlab.com/gitlab-org/gitlab-ce/commit/12818c20430bec5b54903e19ff78b0575c61f05c)

**Comment by hexandcube** [comment]
bump!

**Comment by hexandcube** [comment]
+1

**Comment by hexandcube** [comment]
https://gitlab.com/gitlab-org/gitlab-ce/issues/60810

**Comment by xkr47** [comment]
Yeah I agree with @hexandcube, I would vote for implementing support in the gitlab side instead of keybase. This would allow all basically gitlab instances to work, not just the one at gitlab.com.
